try {
  throw new Error("This is an error");
} catch (error) {
  console.error(error);
}

console.log({
  Hello: "World",
});

console.log(12 + 12);
